src/common/plugin.ts

```diff
@@ -47,10 +49,58 @@ export abstract class Plugin implements IPlugin {
 
     const response = this.container.channelService.hasPermission(channelName, this.permission);
     if (!response) {
-      message.reply(`Please use this command in a \`${this.permission}\` channel`);
+      const baseReply = `Please use this command in a \`${this.permission}\` channel.`;
+
+      if (this.permission.toString() === 'Private') {
+        message.reply(
+          `${baseReply} This is primarily the class channels, and any channels we haven't defined.`
+        );
+        return response;
+      }
+
+      const channels = Object.values(Constants.Channels[this.permission]);
+      const totalChannels = channels.length;
+      channels.splice(10);
```

Comment:
magic number, also is 10 really necessary? That's quite a long error message returned to the user
![image](https://user-images.githubusercontent.com/39809146/112638328-857dbe80-8e15-11eb-8fae-57ec4efb0ffb.png)